Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -34,10 +37,12 @@ public sealed class DisposalUnitSystem : SharedDisposalUnitSystem
     {
         [Dependency] private readonly IMapManager _mapManager = default!;
         [Dependency] private readonly IRobustRandom _robustRandom = default!;
+        [Dependency] private readonly IEntityManager _entMan = default!;
```

Comment:
Entity Systems already have EntityManager so injecting this dependency isn't needed.